Num-Plate-Detector/number_plate.py

```diff
@@ -0,0 +1,82 @@
+# importing libraries
+import numpy as np
+import cv2
+import  imutils
+import sys
+import pytesseract
+import pandas as pd
+import time
+
+image = cv2.imread('car.jpeg')
+
+image = imutils.resize(image, width=500)
+
```

Comment:
```suggestion

pytesseract.pytesseract.tesseract_cmd = input("Enter the path of tesseract in your local system")

```

Like this

Reply:
Now?

![np](https://user-images.githubusercontent.com/73488906/112475367-c1257380-8d96-11eb-9117-7c496ae5eba1.png)

Reply:
@mehabhalodiya your path is incorrect

Reply:
your os is windows??

Reply:
Yes, but not able to find correct path on Ubuntu!

Reply:
you don't have to find correct path for ubuntu

Reply:
enter the tesseract path for your local system on terminal

Reply:
> you don't have to find correct path for ubuntu

Then, what to do?

Reply:
for my local system it was : C:\\Program Files\\Tesseract-OCR\\tesseract.exe

Reply:
and it worked also, without giving an error

Reply:
> for my local system it was : C:\Program Files\Tesseract-OCR\tesseract.exe

Yes I know this for windows but not knowing what to do for ubuntu? What to type?

Reply:
you dont have to check for every type of os

Reply:
Also you can't check for ubuntu being a windows user

Reply:
taking dynamic input is acutallly one of the reasons that every user is able to use code according to theirs system requirements

Reply:
Okay, got it! I am adding this one line in PR.